fluentd/fluentd-gcp-image/Gemfile

```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org'
 
-gem 'fluentd', '~>0.12.32'
-gem 'fluent-plugin-record-reformer', '~>0.8.3'
+gem 'fluentd', '~>0.14.17'
```

Comment:
We distribute 0.14.14, but I think that's just because that was the latest available... Just an FYI.